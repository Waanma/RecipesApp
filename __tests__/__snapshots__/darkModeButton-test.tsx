import React from "react";
import renderer from "react-test-renderer";
import DarkModeButton from "../../components/darkModeButton";

describe("<DarkModeButton />", () => {
  it("renders correctly and matches snapshot", () => {
    const tree = renderer.create(<DarkModeButton />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
