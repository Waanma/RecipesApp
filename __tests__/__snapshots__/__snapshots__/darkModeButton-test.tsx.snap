// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<View
  style={
    [
      {
        "width": "100%",
      },
      {
        "paddingTop": 12,
      },
    ]
  }
>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    testID="darkModeButton"
  >
    <View
      style={
        [
          {
            "marginLeft": 24,
          },
          {
            "width": 48,
          },
          {
            "height": 48,
          },
          {
            "alignItems": "center",
          },
          {
            "justifyContent": "center",
          },
          {
            "borderBottomLeftRadius": 9999,
            "borderBottomRightRadius": 9999,
            "borderTopLeftRadius": 9999,
            "borderTopRightRadius": 9999,
          },
          {
            "backgroundColor": "#00aae4",
          },
        ]
      }
    >
      <Text />
    </View>
  </View>
</View>
`;

exports[`<DarkModeButton /> renders correctly and matches snapshot 1`] = `
<View
  style={
    [
      {
        "width": "100%",
      },
      {
        "paddingTop": 12,
      },
    ]
  }
>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    testID="darkModeButton"
  >
    <View
      style={
        [
          {
            "marginLeft": 24,
          },
          {
            "width": 48,
          },
          {
            "height": 48,
          },
          {
            "alignItems": "center",
          },
          {
            "justifyContent": "center",
          },
          {
            "borderBottomLeftRadius": 9999,
            "borderBottomRightRadius": 9999,
            "borderTopLeftRadius": 9999,
            "borderTopRightRadius": 9999,
          },
          {
            "backgroundColor": "#00aae4",
          },
        ]
      }
    >
      <Text />
    </View>
  </View>
</View>
`;
